### ISSUE TITLE
Parallelize Generator

### ISSUE THREAD

**Issue by hdgarrood** [purescript/pursuit#48]
I'm pretty sure the current generator works sequentially, and spends a lot of time blocked on IO. If we build each package's database in parallel we might be able speed up the generator significantly.

**Comment by hdgarrood** [comment]
No longer relevant; the generator doesn't exist any more.
